src/libs/actions/Policy/Policy.ts

```diff
@@ -2591,6 +2592,22 @@ function createWorkspaceFromIOUPayment(iouReport: OnyxEntry<Report>): WorkspaceF
         value: {[movedReportAction.reportActionID]: null},
     });
 
+    // We know that this new workspace has no BankAccount yet, so we can set
+    // the reimbursement account to be immediately in the setup state for a new bank account
+    successData.push({
```

Comment:
I think we would add this to optimistic too and then remove in failure data

Reply:
good call, thanks!